src/features/listings/components/ListingPage/Comments.tsx

```diff
@@ -84,6 +127,28 @@ export const Comments = ({ refId, refType }: Props) => {
     }
   };
 
+  useEffect(() => {
```

Comment:
what's going on here?

Reply:
whenever use makes a submission, we want a comment in their name automatically
comment goes through but doesn't reflect instantly when scrolled down to the comments.

i have a custom event dispatcher where user submission is done, when the event `update-comments` is dispatched, we listen here and getComments again.

better this way, no downside, do not have to send all comments logic back to page and perform a huge prop drilling.